app/views/changeset/_tab_list.html.erb

```diff
@@ -0,0 +1,7 @@
+<ul class="nav nav-tabs">
+  <% Changeset::ATTRIBUTE_TABS.each do |tab| %>
+    <li class="<%= active_changeset_tab(tab, active) %>">
```

Comment:
risks is hardcoded in the other partial and not here ... looks like one of them is wrong ?

Reply:
After a little playing around, I'm thinking of removing this, since there's already tab-switching logic (including default 'active' class) in `deploys.js`. What do you think?

Reply:
sounds good